sale_fixed_discount/models/sale_order_line.py

```diff
@@ -15,6 +15,13 @@ class SaleOrderLine(models.Model):
         help="Fixed amount discount.",
     )
 
+    @api.model
```

Comment:
```suggestion
    @api.model_create_multi
```

And change the function accordingly

Reply:
You are right, done

Reply:
@sefirosweb Of course, the implementation of the function should be changed accordingly :sweat_smile: